_includes/events/table-events.md

```diff
@@ -6,5 +6,5 @@ The following events are provided by the [Table plugin]({{ site.baseurl }}/plugi
 | newcell | N/A | Fired when a cell is created |
 | ObjectResizeStart | N/A | Fired when a resize action is started on a table, row, column or cell using the resize bars |
 | ObjectResized | N/A | Fired when a resize action is finished on a table, row, column or cell using the resize bars |
-| TableModified | `{ table: HTMLTableElement }` | Fired when style or structural changes are made to a table. <br/>{{site.requires_5_6v}} |
-| TableSelectionClear | N/A  | Fired when the table selection is cleared. <br/>{{ site.requires_5_1v }} |
\ No newline at end of file
+| TableModified | `{ table: HTMLTableElement, structure: boolean, style: boolean }` | Fired when style or structural changes are made to a table. <br/>{{site.requires_5_6v}} |
```

Comment:
Do we need to say this requires 5.7 now? If not maybe `structure` and `style` should be marked as optional?

Reply:
My personal preference would be to go with 5.7

Reply:
Yeah, I was leaning that way as well. @tylerkelly13 thoughts?

Reply:
this is why we need better versioning of the docs....
let's bump it to 5.7.